index.js

```diff
@@ -146,6 +129,27 @@ function processFile (file, config, watcher) {
   })
 }
 
+function config () {
+  if (argv.use || argv.parser || argv.stringifier || argv.syntax) {
+    return {
+      plugins: argv.use ? argv.use.map(i => require(i)) : [],
+      options: {
+        parser: argv.parser ? require(argv.parser) : undefined,
+        syntax: argv.syntax ? require(argv.syntax) : undefined,
+        stringifier: argv.stringifier ? require(argv.stringifier) : undefined,
+        map: argv.map
+      }
+    }
+  } else {
+    return postcssrc()
+    .catch(e => {
```

Comment:
Then my linter f** up ) I was wondering, the `{ return err}` are needed , anyways just cosmetics :)